### DISCUSSION TITLE
comparison plots of different experiments

### DISCUSSION THREAD

**Discussion by BBEK-Anand** [BBEK-Anand/PyTorchLabFlow#3] [General]
x = performance_plot(figsize=(10,3),metric=['val_accuracy',"train_loss","val_loss"],)

![download (5)](https://github.com/user-attachments/assets/fb9623b1-aa8f-4c2e-8fb2-21a00ccf378d)

![download (4)](https://github.com/user-attachments/assets/52647d2d-8caf-49ab-8e8b-5a077486332a)

![download (5)](https://github.com/user-attachments/assets/bea5dea8-b7fc-465d-baa3-72268d071386)
